Templates/GitlabCIPipeline/Reporting-pipeline/wazi-deploy-query/renderer.yml

```diff
@@ -0,0 +1,31 @@
+{% if type is defined -%}
```

Comment:
Line 1 - 29 seems to be not necessary, in my opinion. The variables are always containing a either a value or `*` as per the default in the pipeline variables .